env/env.go

```diff
@@ -38,6 +38,7 @@ const (
 	saveExecutionResult = "save_execution_result"
 	// CsvDelimiter holds delimiter used to parse csv files
 	CsvDelimiter                   = "csv_delimiter"
+	allowCaseInSensitiveTags       = "allow_case_insensitive_tags"
```

Comment:
For readabioity better to call this 
```suggestion
	allowCaseSensitiveTags       = "allow_case_sensitive_tags"
```

And set it as false by default.

Reply:
I'm confused @zabil. Do we want to enable case sensitivity by default?

Reply:
Done.